src/recipes/is-boolean.recipe.ts

```diff
@@ -22,6 +23,16 @@ export function IsBooleanRecipe<Options>(
         // convert strings to boolean
         TypePropertyDecorator(() => Boolean),
 
+        // convert 'false' and '0' to false
+        TransformPropertyDecorator(({ obj, key, value }: TransformFnParams) => {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+            if (obj[key] === 'false' || obj[key] === '0') {
```

Comment:
I would use a case-insensitive comparison here

Reply:
Also, any reason you have to compare `obj[key]` instead of `value` ?

Reply:
value is already set to true when obj[key] has the string value "false"